tests/scripts/check-names.sh

```diff
@@ -16,12 +16,22 @@ if grep --version|head -n1|grep GNU >/dev/null; then :; else
     exit 1
 fi
 
+trace=
+if [ $# -ne 0 ] && [ "$1" = "-v" ]; then
```

Comment:
Minor: As we are adding a new parameter to the CLI, we may consider documenting it somehow as learning that `check-names.sh` takes a `-v` argument will require studying the soruce.

Reply:
I agree, but this is a quick patch to help diagnose a CI issue to a script that we want to replace when we get a minute, so I'd rather not spend time making it fully clean.